eventmesh-connectors/eventmesh-connector-file/build.gradle

```diff
@@ -17,6 +17,8 @@
 
 dependencies {
     api project(":eventmesh-openconnect:eventmesh-openconnect-java")
+    implementation 'org.junit.jupiter:junit-jupiter:5.8.1'
+    implementation 'org.mockito:mockito-junit-jupiter:5.3.1'
```

Comment:
Please change `implementation` to `testImplementation`. Also, there is no need to specify the version number of these 2 jars additionally.